docs/master/api-reference/directives.md

```diff
@@ -142,6 +228,24 @@ type Mutation {
 }
 ```
 
+### Definition
+
+```graphql
+directive @broadcast(
+  """
+  Name of the subscription.
+  """
+  subscription: String
+  
+  """
+  Specify whether or not the job should be queued.
+  """
+  shouldQueue: Boolean
```

Comment:
Do we have a default here?

Reply:
There is not a default value chosen in the `directiveArgValue()`, but the variable is passed on to `Subscription::broadcast()` which defaults `$shouldQueue = null`

Reply:
Alright then, so it is a dynamic default value. Let's add it in the description.